docs/en/remote-flow.rst

```diff
@@ -329,29 +367,21 @@ The JWT payload parameters are described herein:
     - String determining the HTTP method to be used with the `request_uri` endpoint to provide the Wallet Instance metadata to the Relying Party. The value is case-insensitive and can be set to: `get` or `post`. The GET method, as defined in [@RFC9101], involves the Wallet Instance  sending a GET request to retrieve a Request Object. The POST method involves the Wallet Instance requesting the creation of a new Request Object by sending an HTTP POST request, with its metadata, to the request URI of the Relying Party.
```

Comment:
I think that the `request_uri_method` should be removed from the request object. The Wallet has already performed the GET/POST request to the Request URI to obtain the Request Object, it does not need this info here.

Reply:
the way we have included it in the qrcode is pretty custom and in the form of an hint to anticipate its use and then to have a more efficient flow

The place of the request_uri_method is in the request response object